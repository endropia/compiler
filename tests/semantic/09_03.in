var
	a: array[0..10] of integer;

begin
	write(a[0.1]);
end.