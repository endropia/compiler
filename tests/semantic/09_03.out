(5, 8) Integer expected in parameter